src/main/java/org/jboss/query/Predicate.java

```diff
@@ -33,7 +33,7 @@ public String apply(@Nonnull String lhs) {
                     "Predicate %s can't be used only with a single operand".formatted(this.value.strip()));
         }
 
-        return "%s %s ".formatted(value, lhs);
+        return "%s %s ".formatted(lhs, value);
```

Comment:
lol

Reply:
I believe JVM anyway compiles these codes into `return lhs + " " + value` so no degradation should happen